guides/common/modules/proc_adding-images-to-server.adoc

```diff
@@ -54,10 +54,24 @@ Use the `--uuid` field to store the full path of the image location on the KVM s
 --uuid "/var/lib/libvirt/images/_KVMimage_.qcow2" \
 ----
 endif::[]
+ifeval::["{context}" == "rhv-provisioning"]
```

Comment:
Why not ifdef here?

Reply:
Didn't seem worth changing all the others when it works the same for a long block :shrug: 

I'm happy to change them in the next PR though, if you want to switch to ifdefs as much as possible.

Reply:
Ah I did not get that it was there before. I am fine with ifevals as they just work.